src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -32,6 +32,8 @@
 public class PromregatorApplication {
 	
 	private static final Logger log = Logger.getLogger(PromregatorApplication.class);
+
+	public static final String SPACE_ALL_APPLICATIONS = "SPACE_ALL_APPLICATIONS";
```

Comment:
As this constant is related only to `Target#applicationName`, I think it would be better to have that constant at `Target`.
This would also prevent that we have to `import org.cloudfoundry.promregator.PromregatorApplication` in `ReactiveCFAccessorImpl` and further classes too, which also violates the hierarchical class structure.

Reply:
+1